setup.py

```diff
@@ -0,0 +1,41 @@
+import os
+
+from setuptools import setup, find_packages
+
+with open('README.rst', 'rb') as f:
+    LONG_DESCRIPTION = f.read().decode('utf8')
+
+# Single source the version.
+version_file = os.path.realpath(os.path.join(
+    os.path.dirname(__file__), 'pymongoexplain', 'version.py'))
```

Comment:
Can you add the version.py file? It should look like this but the version should be "0.1.0.dev0": https://github.com/mongodb/pymongo-auth-aws/blob/master/pymongo_auth_aws/version.py